File: PRAK304-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
for i in range(0, 5):
    n = int(input())
    if n >= 100 or n < 0:
        print("Anda Menginput Melebihi Limit Bilangan")
    elif n >= 20:
        print("Puluhan")
    elif n >= 10:
        print("Belasan")
    elif n >= 1:
        print("Satuan")
    else:
        print("Nol")

File: PRAK304-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>
int main(){
    int n;
    for(int i = 0; i < 5; i++){
    scanf("%d", &n);
    if(n >= 100 || n < 0){printf("Anda Menginput Melebihi Limit Bilangan\n");}
    else if (n >= 20){printf("Puluhan\n");}
    else if (n >= 10){printf("Belasan\n");}
    else if (n >= 1 ){printf("Satuan\n");}
    else{printf("Nol\n");}
    }
    return 0;
}